content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -9,4 +9,345 @@ owner: istio/wg-networking-maintainers
+
+To send some requests to the `reviews` service via the `productpage` service from the `sleep` pod:
+
+{{< text bash >}}
+$ kubectl exec deploy/sleep -- curl -s http://productpage:9080/productpage
+{{< /text >}}
+
+To send some requests to the `reviews` `v2` pod from the `sleep` pod:
+
+{{< text bash >}}
+$ export REVIEWS_V2_POD_IP=$(kubectl get pod -l version=v2,app=reviews -o jsonpath='{.items[0].status.podIP}')
+$ kubectl exec deploy/sleep -- curl -s http://$REVIEWS_V2_POD_IP:9080/reviews/1
+{{< /text >}}
+
+Requests to the `reviews` service should be enforced by the `reviews-svc-waypoint` for any L7 policies.
+Requests to the `reviews` `v2` pod should be enforced by the `reviews-v2-pod-waypoint` for any L7 policies.
+
+1.  If your L7 policy isn't enforced, run `istioctl analyzer` first to check if your policy has any validation issue.
+
+    {{< text bash >}}
+    $ istioctl analyze
+    ✔ No validation issues found when analyzing namespace: default.
+    {{< /text >}}
+
+1.  Determine which waypoint is enforcing the L7 policy for your service or pod.
+
+    If your source calls for the destination as its service's hostname or IP, use the `istioctl experimental ztunnel-config service` command to confirm your waypoint is used by the destination service. Following the example earlier, the `reviews` service should use the `reviews-svc-waypoint` while all other services in the `default` namespace should use the namespace `waypoint`.
+
+    {{< text bash >}}
+    $ istioctl experimental ztunnel-config service
+    NAMESPACE    SERVICE NAME            SERVICE VIP   WAYPOINT
+    default      bookinfo-gateway-istio  10.43.164.194 waypoint
+    default      bookinfo-gateway-istio  10.43.164.194 waypoint
+    default      bookinfo-gateway-istio  10.43.164.194 waypoint
+    default      bookinfo-gateway-istio  10.43.164.194 waypoint
+    default      details                 10.43.160.119 waypoint
+    default      kubernetes              10.43.0.1     waypoint
+    default      notsleep                10.43.156.147 waypoint
+    default      productpage             10.43.172.254 waypoint
+    default      ratings                 10.43.71.236  waypoint
+    default      reviews                 10.43.162.105 reviews-svc-waypoint
+    ...
+    {{< /text >}}
+
+    If your source calls for the destination using pod IP , use the `istioctl experimental ztunnel-config workload` command to confirm your waypoint is used by the destination pod. Following the example earlier, the `reviews` `v2` pod should use the `reviews-v2-pod-waypoint` while all other pods in the `default` namespace should not have any waypoints as by default only services use the namespace `waypoint`.
```

Comment:
```suggestion
    If your source calls the destination using a pod IP , use the `istioctl experimental ztunnel-config workload` command to confirm your waypoint is used by the destination pod. Following the example earlier, the `reviews` `v2` pod should use the `reviews-v2-pod-waypoint` while all other pods in the `default` namespace should not have any waypoints because by default only services use the namespace `waypoint`.
```